### ISSUE TITLE
Tensorboard 404 Errors

### ISSUE THREAD

**Issue by jubjamie** [tensorflow/tensorflow#9976]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N/A
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Win 10
- **TensorFlow installed from (source or binary)**: pip
- **TensorFlow version (use command below)**: 1.1.0
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:5.1
- **GPU model and memory**:k40 12gb
- **Exact command to reproduce**: launch tensorboard

### Describe the problem
Tensorboard is 404'ing on a lot of resources. See the error messages below. I think I saw a similar issue somewhere so this may be a duplicate. You're probably aware but thought i'd file just in case.

### Source code / logs
```
WARNING:tensorflow:path ../external\weblas_weblas_js/file/weblas.map.json not found, sending 404
WARNING:tensorflow:path ../external\web_animations_js/web-animations-next-lite.min.js.map not found, sending 404
WARNING:tensorflow:path ../external\weblas_weblas_js/file/weblas.map.json not found, sending 404
WARNING:tensorflow:path ../external\web_animations_js/web-animations-next-lite.min.js.map not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
```

**Comment by skye** [comment]
@dandelionmane @jart

**Comment by jart** [comment]
We're working on the text/plugin 404. The other ones are for js.map files and shouldn't cause problems if they're not available.

**Comment by jubjamie** [comment]
Ok. I guess i'll leave this open until it's sorted then. Thanks.

**Comment by lincolnBush** [comment]
Get the same problem with this system information:
Ubuntu 16.04
python 3.5.2
tensorflow 1.1.0

**Comment by XeMinZa** [comment]
I have the same problem! how can I fix it?

**Comment by jart** [comment]
@XeMinZa I'm assuming you're on windows? Can you open a new bug for that and CC me?

**Comment by jart** [comment]
@XeMinZa I'm assuming you're on Windows?

**Comment by XeMinZa** [comment]
Yes, I'm using win10. Someone said that this bug was fixed in the later release. But I still have it...

**Comment by jubjamie** [comment]
@XeMinZa what exact Tensorflow version? I'm on Windows and I think this bug has gone. I can double check though.

**Comment by hongjiedai** [comment]
I use v1.1.0 and meet the similar issue.

Starting TensorBoard b'47' at http://0.0.0.0:6007
(Press CTRL+C to quit)
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404

**Comment by 1042799521** [comment]
I have the same problem!!!
Mac     python2.7    running in virtualenv

**Comment by leckie-chn** [comment]
Same problem.
tf 1.1.0 + Ubuntu 16.04 + python 3.5 + cuda 8.0

**Comment by astrung** [comment]
Same issue with : tf 1.1.0 + Ubuntu 16.04 + python 3.5 + cuda 8.0. I can not access it from internet

**Comment by jart** [comment]
I've confirmed that all these 404 warning messages are now gone at HEAD. They should be relatively harmless in the old releases.

**Comment by jubjamie** [comment]
Cheers @jart

**Comment by Maaater** [comment]
I have the same problem when i use 
c:> tensorboard --logdir='DIR'
but when i use
c:> tensorboard --logdir= "DIR"
its ok
win10

**Comment by sivaram10** [comment]
Still facing this issue.
System Info:
OS : Windows 10
Anaconda3
Tensorflow 1.1.0 (installed via Anconda navigator)
Starting TensorBoard b'47' at http://0.0.0.0:6006
(Press CTRL+C to quit)
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404
WARNING:tensorflow:path ../external\data/plugin/text/runs not found, sending 404

**Comment by jubjamie** [comment]
@sivaram10 Can you please try upgrading Tensorflow. As you are on an old version then you won't have the fixes released in newer version. I believe you may have to install via pip instead of conda to receive the latest version.

**Comment by sivaram10** [comment]
Thanks @jubjamie  I updated Tensorflow to 1.2.1 version. I'm not getting those warnings anymore. But, I'm still not able to see graphs and scalar data. I can see the log files.

**Comment by jubjamie** [comment]
I would suggest refiling over at the **tensorboard** issue tracker as this 404 issue is now closed.

**Comment by Ladyzhu** [comment]
I once opened a file successfully, but after a while, when I opened it again, it would show this error.Why?please give me some help

**Comment by jubjamie** [comment]
I would suggest refiling over at the **tensorboard** issue tracker.

**Comment by zhangwlgq** [comment]
> I have the same problem when i use
c:> tensorboard --logdir='DIR'
but when i use
c:> tensorboard --logdir= "DIR"
its ok
win10

it helps!

**Comment by gamijin0** [comment]
Do you have a  directory named "data" beside the log-dir? 
When I rename the "data" dir, my problem fixed.

**Comment by yangtao4389** [comment]
I updated chrome and its work

**Comment by tartavull** [comment]
It doesn't work on chromium nor firefox for me.

**Comment by joellerena** [comment]
I have the same error:
WARNING: tensorflow: path ../external/data/plugin/text/runs not found, sending 404

I have tensorflow 1.1.0 and tensorboard 0.1.8.
I read: https://github.com/tensorflow/tensorboard/issues/743

I reinstalled 1.1.0 and it worked, but it showed me in:
- tensorboard --logdir .....
- Starting TensorBoard 47 at http://0.0.0.0:6006
incompatibility with numpy as warnings

I have updated tensorboard to 1.11.0
I have updated numpy with sudo pip install --upgrade numpy
it works again but with this 404 message.

Should I download to a smaller version of Tensorboard?

Ubuntu: 16.04
Python 2.7
numpy 1.15.2
tensorflow 1.1.0
tensorboard 1.11.0

I appreciate your help
